src/test/java/uk/gov/cshr/controller/signup/SignupControllerTest.java

```diff
@@ -226,7 +227,7 @@ public void shouldRedirectToEnterTokenPageIfInviteNotAuthorised() throws Excepti
                 get("/signup/" + code)
                         .with(CsrfRequestPostProcessor.csrf()))
                 .andExpect(status().is3xxRedirection())
-                .andExpect(redirectedUrl("/signup/enterToken/" + code));
+                .andExpect(redirectedUrl("/signup/request"));
```

Comment:
This expect is not in synch with the unit test method name. Method name suggests that expectation is to redirect to "Enter Token Page" rather than signup. Please either correct the method name or correct the expectation.

Reply:
DONE.
I have change the test method to reflect the current workflow (functionality changes). Please refer to the latest changes in the file.